sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSiteReview.md

```diff
@@ -0,0 +1,116 @@
+﻿---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
+schema: 2.0.0
+author: pullabhk
+ms.author: pullabhk
+manager: 
+ms.reviewer:
+---
+
+# Get-SPOSiteReview
+
+## SYNOPSIS
+Track all site access reviews initiated by all SharePoint administrators in Data Access Governance (DAG) reports.
```

Comment:
>SharePoint administrators [](http://example.com/codeflow?start=47&length=25)

official name is uppercase "A" - Administrator

Why must SharePoint Administrator be called out specifically? How about Global Administrators? Maybe just "administrators" (generic, not referencing anyone in particular) is the better wording?

Reply:
We are expecting SharePoint admins to use this feature in SharePoint admin center and using these PS commands. Hence the specification.

Reply:
in that case remove the line.

Only SharePoint admins can use these cmdlets; specifying is being redundant for no reason